flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -2051,6 +2063,40 @@ static void genStandaloneDo(lower::AbstractConverter &converter,
                 llvm::omp::Directive::OMPD_do, dsp);
 }
 
+static void genLoopOp(lower::AbstractConverter &converter,
```

Comment:
Ultra-nit: This doesn't belong in this section of the file, since the `loop` construct never belongs to a composite construct. It should go below `genLoopNestOp`.